framework/Optimizers/Optimizer.py

```diff
@@ -400,7 +400,7 @@ def _initializeInitSampler(self, externalSeeding):
                                    .format(v=sampled, i=self._initSampler.name, s=self.name))
     self._initSampler.initialize(externalSeeding)
     # initialize points
-    numTraj = len(self._initialValues) if self._initialValues else None
+    numTraj = len(self._initialValues)
```

Comment:
Is this part of a different pull request?

Reply:
I am sorry. Reverted